ts/store/reducers/backendStatus/remoteConfig.ts

```diff
@@ -393,19 +394,37 @@ export const landingScreenBannerOrderSelector = (state: GlobalState) =>
   );
 
 /**
- * The 'disabledForMessagesAndServices' on the 'ioMarkdown' is used to
- * remotely disable the IO Markdown feature for messages and services,
- * bypassing the local feature flag (in the relative selector).
- * @param state The global redux state
- * @returns true if IOMarkdown has to be disabled for messages and services
+ * This selector is used to know if IOMarkdown is enabled on Messages and Services
+ *
+ * @returns true (enabled) if:
+ * - the IOMarkdown configuration is missing
+ * - the min_app_version parameter is missing
+ * - current app version is greater than or equal to the min app version
+ * false (disabled) if:
+ * - CDN data is not available
+ * - current app version is lower than the min app version
+ * - min app version is set to 0
```

Comment:
Thinking about this logic, there's a dark path that I guess we're not considering: if we find a bug in the new IO Markdown component during the internal testing period, when we'll remove both selector and remoteConfig, old versions of the app (ones with the bug) would have the component enabled by default. So my suggestion is moving  this 2 cases:
```
- the IOMarkdown configuration is missing
- the min_app_version parameter is missing
```
to disable the component in app and remove by steps the selector and than the remoteConfig in io-services-metadata in future

What do you think?

Reply:
100% clear by any doubt!